### DISCUSSION TITLE
inspired by Agenda (iOS): calendar-focused journal-linking and views + linked tags + converting entries

### DISCUSSION THREAD

**Discussion by steinhmor** [TechbeeAT/jtxBoard#92] [Ideas]
After talking to the jtx-Team on Mastodon I realised, that this meight be the better place for this suggestions 😉

jtx already seems to be a fantastic app and I wish you the very best for the upcoming year(s)! Many the following ideas could find their way into the backlog and I would be happy to help testing and give feedback if needed!

## In short:
Agenda is an interesting App for iOS that might be inspiring in some points. Here are some ideas (partly inspired by Agenda) that would be great to have for jtx.
- Linking a note to an calendar-event. This would speed up the process of selecting a date and time manually. 
- Calendar-view for Notes / to do / journal: like the kanban-view it would be great to have a column-based calendar-view based on the journal-date or the due-date for todos - or even combined? (which is probably only useful with many entries...)
- linking tags like Resources and Categories to a pre-filtered list wich contains entries with similar tags.
- option to convert a note into a to do or journal or vice versa.

## Long version in German from [Mastodon](https://det.social/@steinhmor/109586040987832829) :
Ich bin heute auf eure tolle App gestoßen, als ich nach einer Android-Alternative zu Agenda (für iOS) gesucht habe. Ich bin beeindruckt und gespannt, was ihr noch alles vorhabt. Gibt es Pläne, ähnlich wie bei Agenda Notizen direkt mit Termineinträgen zu verknüpfen? Dieses Konzept finde ich nämlich sehr interessant und praktisch - momentan würde ich dieses Konzept mit Journaleinträgen mit Daten in der Zukunft nachbauen, eine direkte Auswahl eines Termins ginge aber schneller 😀
konkretes Anwendungsbeispiel ist die Nutzung einer APP zur Unterrichtsplanung. Für jede Unterrichtsstunde existiert ein Kalendereintrag. Mit Agenda ist dann das 'dranhängen' einer Notiz (= Unterrichtsplanung) sehr einfach, die Übersichtlichkeit könnte aber noch verbessert werden... Mit jtx konnte ich das in Teilen schon nachbauen (sehr praktisch finde ich auch 'Material' und weitere Felder 😉 - hier wäre eine Verknüpfung der Tags zu einer entsprechend gefilterten Liste Hammer)

**Comment by patrickunterwegs** [comment]
Hey @steinhmor , thanks again 😉

For now I want to pick out two smaller things:

> linking tags like Resources and Categories to a pre-filtered list wich contains entries with similar tags.

In jtx Board 1.x there was the option in the detail view to click on a category or resource and it would take you back to the list with all entries filtered by that category or resource. (I actually didn't implement it in jtx Board 2.x as I thought nobody used it 😅) but would you say this is what fulfills your request?

> option to convert a note into a to do or journal or vice versa.

There is already the option to copy an entry as a journal, note or task. Would you say you require the same thing but to transform the entry instead of copying it?

**Reply by steinhmor** [reply]
> In jtx Board 1.x there was the option in the detail view to click on a category or resource and it would take you back to the list with all entries filtered by that category or resource.

Yeah, that sounds intuitive. I would expected the categories to be clickable because off the ui-design 😉

> There is already the option to copy an entry as a journal, note or task.

Uuups - i did not notice this option. Fits perfectly for me - as deleting the old entry is no big thig if it is needed. But maybe it would be interesting to directly link the entries to each other? Which leads me to another question: I expected the linked notes section to be an autocompleted field which allows me to link one entry to another existing one (like Joplin is able to link notes - but a bit easier without an ID...). Is this possible or planed (or did I just missed another already implemented feature 😄)?

**Reply by patrickunterwegs** [reply]
Alright, so for the categories and resources I have created #100 😉 

Linking existing entries to another existing entry is not implemented yet. This might come in the future, but for now there are still other priorities, unless enough people convince me to do it earlier 😉
